lua/core/arc.lua

```diff
@@ -92,6 +93,46 @@ function Arc:refresh()
   monome_refresh(self.dev)
 end
 
+--- draw anti-aliased point to point arc
```

Comment:
Reviewing docs and this is not entirely clear to me and does not match other LED descriptions.

Should this read as "sets the state of a contiguous group of LEDs on a single ring of this arc device - between two angle values" ?

What does "anti-aliased" mean in this context? Is there a better description we can add to the lua-docs?

Reply:
anti-aliased means the point can be a decimal, in between LEDs. it will do a nice fade.

point-to-point "arc" as in segment/range on the circle. perhaps it seems confusing because the name of the device is arc, as is the technical term for a segment of a circle?

Reply:
OK - should I add those descriptions (or something better) to the api docs?